corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -35,3 +47,26 @@ def test_files_match_modules(self):
 
         if errors:
             self.fail("Mismatched JS file/modules: \n{}".format("\n".join(errors)))
+
+    def test_requirejs_disallows_hqimport(self):
+        errors = []
+        for filename in self.requirejs_files:
+            # Special case 1: ignore standard_hq_report.js until we migrate UCRs and reports
+            if filename.endswith("reports/js/standard_hq_report.js"):
+                continue
+            # Special case 2: knockout_bindings should be broken up, in the meantime, ignore
+            if filename.endswith("hqwebapp/js/knockout_bindings.ko.js"):
+                continue
+
+            proc = subprocess.Popen(["grep", "hqImport", filename], stdout=subprocess.PIPE)
```

Comment:
Might it be worth spinning the proc off into a greenlet pool or something so it isn't blocking

Reply:
720f2bd